### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by LegendarySpork9** [FINDarkside/TLD-Save-Editor-3#12]
Hello Matey,

I hope this finds you well.

I have been a fan of this program for a long time and have always used it to get me out of a jam.

I read your comment about this potentially not being updated/being abandoned. As a guy who codes for fun I can understand how long it takes and wanting to move onto other things.

I was prepared to leave it at that when I notice that here you made changes that suggest you might be working on it?

Just curious as to whether you are continue with TLD Save Editor 3? If so, any idea on a release?

Hope this doesn't come across as pushy, I am just curious :).

Cheers,

LS9

**Comment by FINDarkside** [comment]
Hi, no worries you're not being pushy! 

I'm not really sure if I'll continue on this to be honest. I took a quick look of the new TLD update and the changes to save format don't seem to be big. However I am working full time now and don't have that much interest in personal projects right now. I hope I can get to finish this to usable state at some point, but no promises.

**Comment by LegendarySpork9** [comment]
No worries mate, I work full time too and its difficult to balance everything.

What program do you use to code? Perhaps I could cast my eyes over it and see if I can get it to work on my down time. I can set aside some time for it as I have some days off coming up. Just a suggestion.

**Comment by FINDarkside** [comment]
I use VS Code. I assume you noticed but this new project is TypeScript + Vue + Electron and the old one is C# + WPF. If you want to give it a try that'd be awesome! If you need any help you can contact me here or in Discord  (FINDarkside#1337).

I'll try to write a readme in the next couple days on how to get the project to run etc.

**Comment by Moumirrai** [comment]
Hi, 
I'd like to help with this project and keep it working with new versions of TLD.
Although I've never worked with electron and I'm rather new to Vue, I think I could get the hang of it. 

Would you mind if I occasionally ask for some additional info about your code or opinion?

**Comment by FINDarkside** [comment]
> Would you mind if I occasionally ask for some additional info about your code or opinion?

Not at all! If you use discord you can also add me there (FINDarkside#1337).
